### ISSUE TITLE
Show spinner for annotation download requests

### ISSUE THREAD

**Issue by fm3** [scalableminds/webknossos#6046]
When downloading a large annotation, the request is answered only after a noticeable delay (despite #6036 ). To avoid users clicking multiple times or get frustrated, we may want to show a spinner indicating that the browser is waiting for the download request.
I think something like this already exists in the list views, but not in the annotation view (where download is available from the top bar)

**Comment by hotzenklotz** [comment]
@fm3 Is this still relevant after introducing the new download modal?

**Comment by fm3** [comment]
Yes, I’d say so, the modal closes when the user hits ok, but the download does not start immediately.
